components/Utilities/SharedBus/SharedBus.c

```diff
@@ -0,0 +1,96 @@
+#include <esp_err.h>
+#include "SharedBus.h"
+#include "Custom_Log.h"
+
+#define UI_EVENT_FLAG	( 1 << UI_INTERFACE_ID )
+#define UI_EVENT_FLAG	( 1 << UI_INTERFACE_ID )
+
+#define BIT_23	( 1 << 23 ) //prevent overwriting bus for send before one turn of the event loop
+#define BIT_22	( 1 << 22 ) //peek to receive switch flag 
+#define BIT_21	( 1 << 21 ) //peek to receive switch flag 
+
```

Comment:
please write them in .h file

Reply:
they are all used locally so it's better to keep them here.